preparedata/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/preparedata/osm/OSMGISLayersTests.groovy

```diff
@@ -50,5 +51,23 @@ class OSMGISLayersTests {
 
         //h2GIS.getTable(process.results.hydroTableName).save("./target/osm_hydro.shp")
         assertEquals 10, h2GIS.getTable(process.results.hydroTableName).rowCount
+
+        //h2GIS.getTable(process.results.imperviousTableName).save("./target/osm_hydro.shp")
+        assertEquals 45, h2GIS.getTable(process.results.imperviousTableName).rowCount
+    }
+
+    @Test
+    void createImperviousTableTest() {
```

Comment:
merge this test with the creategislayers test

Reply:
OK, I'll remove it.